### DISCUSSION TITLE
Esta semana...

### DISCUSSION THREAD

**Discussion by mmasias** [mmasias/24-25-IdSw1#129] [Announcements]
Hola a todos

Esta semana, como sabemos, está destinada a la segunda reunión formal con su cliente. Muchos de ustedes han tenido ya más de una, por lo que esta es una más en un flujo de comunicación saludablemente establecido. 

Recordar que la iniciativa está del lado nuestro, es decir, que tanto si hace falta agregar una reunión adicional como si ya están establecidos cronogramas, la iniciativa debe partir de nosotros.

Un buen avance requeriría tener -a día de hoy-:

- [ ] Un modelo del dominio claramente establecido, sin fisuras.
  - [ ] Entidades
  - [ ] Estados
  - [ ] Glosario
- [ ] Actores y casos de uso identificados
  - [ ] Diagrama de contexto para cada grupo de actores y los casos de uso que le competen adecuadamente relacionados en este diagrama.
- [ ] Casos de uso más complejos detallados

¿Lo tenemos así? Si no, conviene apretar...

Estaré, como siempre, disponible para dudas, consultar, tutorias, etc... Ah, y estoy mirando el parcial, a ver si lo dejo resuelto también esta semana.

Manuel

PS: Los **responsables** de grupo, poner un comentario de conformidad de haber leido este mensaje.

**Comment by TheMoys** [comment]
Hola Manuel! 

Las fechas y horarios los proponemos nosotros con nuestro cliente o lo pondrás tú cómo la primera vez?

**Reply by mmasias** [reply]
Ustedes mismos!

**Reply by mmasias** [reply]
Lo de hacerlo durante las horas de clase es la fecha/hora mínima en la que todos coinciden (porque están en clase). Pero si por lo que sea a su cliente le viene mejor otra fecha/hora y todos están de acuerdo, por mi no hay problema.

**Reply by TheMoys** [reply]
Entonces las sesiones de esta semana...? No habrá o como?

**Reply by mmasias** [reply]
> Entonces las sesiones de esta semana...? No habrá o como?

¿Sesiones de clase? No: esta semana es dedicada totalmente a las reuniones con los clientes y un empujón considerable al avance. Durante las horas de clase estaré en el aula (o en mi despacho, es igual), para absolver las dudas que puedan haber.

La siguiente semana haremos una revisión profunda del trabajo. Tomar como referencia lo que les puse arriba, en el post inicial.

**Comment by celiabecerril** [comment]
Leído
